res/photon/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Photon Styling Examples</title>
+    <style>
+      body, html {
+        font: message-box;
+        font-size: 11px;
+      }
+      .row {
+        border-left: 2px solid var(--grey-30);
+        padding: 0 3em 2em;
+        margin: 3em 0;
+      }
+      .row pre {
+        margin: 2em 0;
+        display: block;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="row">
+      <pre>&lt;button type="button" class="photon-button"&gt;Photon Button&lt;/button&gt;</pre>
+      <button type="button" class="photon-button">Photon Button</button>
+    </div>
```

Comment:
can you also add these cases:
* `<button type="button" class="photon-button photon-button-micro">Photon Button</button>`
* `<button type="button" class="photon-button photon-button-ghost"></button>` <= that one should also have a background-image, for example with the sidebar icon (possibly as a data-url if you don't want an external dependency)